__tests__/omnibox-directions.test.js

```diff
@@ -0,0 +1,231 @@
+import React from "react";
+import OmniboxDirections, {
+  getAutoCompleteHeight
+} from "../src/components/search/omnibox-directions.component";
+import renderer from "react-test-renderer";
+import { shallow } from "enzyme";
+import { Platform } from "react-native";
+
+let mockPOIs = [
```

Comment:
BLOCKING
Same comment
See: https://github.com/RGPosadas/WayFinder/blob/TASK-10/setup-detox/__tests__/custom-marker.test.js#L54

Reply:
test converted to typescript